packages/airnode-adapter/src/response-processing/extraction.test.ts

```diff
@@ -247,3 +247,29 @@ describe('extract and encode multiple values', () => {
     });
   });
 });
+
+describe('respects maximum encoded size', () => {
+  it('throws when the encoded value is too large', () => {
+    const str = 'x'.repeat(417);
```

Comment:
The encoded type can be `bytes` so that this number is 1025 and not this magic number

Reply:
Ideally 1024 should be imported from `constants.ts` and this is derived from that